src/Tokens.js

```diff
@@ -0,0 +1,544 @@
+const fs = require('fs')
+const path = require('path')
+const crypto = require('crypto')
+
+const msal = require('@azure/msal-node')
+const XboxLiveAuth = require('@xboxreplay/xboxlive-auth')
+const debug = require('debug')('minecraft-protocol')
+const { SmartBuffer } = require('smart-buffer')
+const jose = require('jose-node-cjs-runtime/jwk/from_key_like')
```

Comment:
1. is it necessary to get the function like this or is it directly exported?
2. instead of using the node runtime, wouldn't it be better to use the general jose package for usage in browser too?

Reply:
From what I understand, thats how we import: https://github.com/panva/jose/blob/HEAD/docs/functions/jwk_from_key_like.fromkeylike.md#function-fromkeylike

Also that is the same package just for CJS not ESM. Thats how the package wants us to use it.

Reply:
The web version of jose is for WebCrypto, we need the node version. Also JwkLike is something that only exists on node.

Reply:
> The web version of jose is for WebCrypto, we need the node version. Also KeyLike is something that only exists on node.

We're going to need to figure something out here.
Jest doesn't like the module name: ```Cannot find module 'jose-node-cjs-runtime/jwk/from_key_like' from 'src/Tokens.js```

Reply:
You can switch it to mocha